crabpy_pyramid/views/adressenregister.py

```diff
@@ -177,7 +177,8 @@ def list_straten(request):
     request = set_http_caching(request, "adressenregister", "short")
     Gateway = request.adressenregister_gateway()
     gemeente_niscode = request.matchdict.get("niscode")
-    straten = handle_gateway_response(Gateway.list_straten, gemeente_niscode)
+    include_homoniem = request.matchdict.get("include_homoniem", False)
```

Comment:
matchdict bevat denk ik enkel parameters die in de route zitten. Ik vermoed dat include homoniem eerder een queryparam zal zijn ?

matchdict
If a [route](https://docs.pylonsproject.org/projects/pyramid/en/latest/glossary.html#term-route) has matched during this request, this attribute will be a dictionary containing the values matched by the URL pattern associated with the route. If a route has not matched during this request, the value of this attribute will be None. See [The Matchdict](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html#matchdict).